CONTRIBUTING.md

```diff
@@ -49,3 +49,15 @@ Now several cases are possible:
    Choose the start branch (the one you want to send your changes to) and the end branch (the one you want to send your changes to).
    Put an understandable name, click on `Assign Yourself` on the right,
    At the bottom of the description, note the id of your github issue like this: `Related issues: #1` or `Close #1`.
+
+## Create new Release
+
+To publish a new release you need to the repository, you need to be sure to be on the latest version. so you can use this commad : 
+
+```sh
+git checkout master && git pull && npm version minor
+```
+
+`git checkout` to be sure that we are on master.
+`git pull` to be sure to have the latest changes on master.
+`npm version` to create the new version. `minor` will increment the latest number. If latest release was `v0.0.9`, so the next will be `v0.1.0`.
```

Comment:
This example is quite confusing.

`minor` resets the patch version, and increments the minor version.

Also it makes a person (at least me) to think it's always `v0.1.0` after `v0.0.9`, but actually `v0.0.10` is correct too.

Reply:
Vhen I've test `npm version minor`, the minor that has automaticaly create the `v0.1.0` instead of creating the `v0.0.10` tag.

Reply:
Yes, it's expected.

Reply:
so wrong...
Just see I use `npm version minor` instead of `npm version patch`